migrate/20241204_vm_enforce_vcpus_memory.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    # Update the memory & vcpus for exisitng VMs, to match the logic from the
+    # code. Then enforce the NOT NULL constraint on the columns.
+    run <<-SQL
```

Comment:
Sequel does not support multiple queries inside a single `run` command.  Whether it works depends on the adapter (so I assume it works with the postgres adapter), but I would advise against such use.  Use separate `run`  methods for the two statements.

Reply:
done